internal/proxy/github.go

```diff
@@ -220,9 +222,7 @@ func (p *Proxy) getUserWithRetry(repositoryCloneURL string, logEntry *log.Entry,
 
 	}
 
-	// Last chance
-	return p.getUser(repositoryCloneURL, logEntry)
-
+	return clients.Namespace{}, errors.New("Couldn't get User, even after retrying " + string(retry) + " times")
```

Comment:
I don't know the code but is clients.Namespace{} return the same as p.getUser?

Reply:
Yup, but this is basically one more retry. If what we want to do is just that, why don't we just increase the retry parameter by 1? It gives takes parameter `retry` and retries for `retry + 1`. Inappropriate.

Reply:
I agree with solving the retry count issue but the error must the be same error as `getUser` How about - 

```go
for i := 1; i < retry; i++ {
  if ns, err := p.getUser(repositoryCloneURL, logEntry); err == nil {
        return ns, nil
  }
  time.Sleep(1*time.Second)
}
return  p.getUser(repositoryCloneURL, logEntry);

Reply:
@sthaha code piece which you are suggesting just makes the code compact and doesn't solve the retry count issue. Anyway, retry count issue will not create any problem, so I will change it to what you are suggesting.